requirements/azure.pip

```diff
@@ -22,20 +22,18 @@ cryptography==42.0.5
     # via
     #   -r requirements/azure.in
     #   azure-storage-blob
-django==3.2.25
+django==4.0
```

Comment:
we should update to 4.2 which is currently in LTS, 4.0 ended security support [11 months ago](https://endoflife.date/django)

Reply:
Oh, yes. I just remembered that my strategy was to go through all the versions but start with 4.0. I will take this back to the draft.